packages/volto/src/components/manage/Blocks/Teaser/DefaultBody.jsx

```diff
@@ -50,16 +51,22 @@ const TeaserDefaultTemplate = (props) => {
             }
           >
             <div className="teaser-item default">
-              {(href.hasPreviewImage || href.image_field || image) && (
+              {url && !isInternalURL(url) ? (
```

Comment:
```suggestion
              {url && !image?.image_field ? (
```
@Tishasoumya-02 try it like this, it will solve the situation where you link directly to a scale from within the site because you simply copied the link from an internal image.

Here is a screenshot showing this in action on my devel
![teaser-url](https://github.com/plone/volto/assets/152852/073b53f0-f759-4c9a-95ef-fceae7e1c4c4)

We also have no test for this interaction where you add as an image override the link to an image scale.

I would be in favor of adding in that test the link to internal images and external image but we still didn't get any feedback from @sneridagh regarding my comment  here https://github.com/plone/volto/pull/6147#discussion_r1663976089